app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  validates :title, length: { maximum: 100 }, presence: true
+  validates :text, length: { maximum: 300 }, presence: true
+
+  after_save :update_post_counter
+
+  def return_last_five_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] Greate job however I would be better to use the active records order method indead of the ruby method eg:


```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```

Reply:
Please remember to specify when a requirement is optional, like this one

Reply:
[OPTIONAL] You are right, I'm sure the previous reviewer intended this to be optional. Great job!!! 💯